src/components/Species/index.tsx

```diff
@@ -40,8 +40,7 @@ import TooltipLearnMoreModal, {
   TooltipLearnMoreModalData,
 } from 'src/components/TooltipLearnMoreModal';
 import PageHeaderWrapper from 'src/components/common/PageHeaderWrapper';
-import PopoverMenu from '../common/PopoverMenu';
-import { DropdownItem, SortOrder } from '@terraware/web-components';
+import { DropdownItem, Popover as PopoverMenu, SortOrder } from '@terraware/web-components';
```

Comment:
Isn't Popover as PopoverMenu confusing here? Because we have the other component that is PopoverMenu

Reply:
I don't think so but I can rename it

Reply:
But why don't we use the Popover name as in other places? Are we using other Popover here?

Reply:
Yes, Popover is also imported :)

Reply:
from mui/material